test_pandalab.py

```diff
@@ -0,0 +1,36 @@
+import scraper
+import logging
+
+from datetime import datetime
+from multiprocessing import Pool, freeze_support
+
+from scraper.pandalab import fetch_slots as pandalab_fetch_slots
+from scraper.pandalab import centre_iterator as pandalab_centre_iterator
+from scraper.pattern.scraper_result import ScraperResult
+from scraper.pattern.scraper_request import ScraperRequest
+
+logger = logging.getLogger('scraper')
+today = datetime.now().strftime('%Y-%m-%d')
+
+def async_pandalab_scrape(centre):
+    vaccine_center = centre["nom"]
+    com_insee = centre["com_insee"]
+    rdv_site_web = centre["rdv_site_web"]
+    logger.info("Vaccine Center {0}: looking for an appointment".format(vaccine_center))
```

Comment:
Si on est parti pour utiliser des f-strings, autant y aller jusqu'au bout :
```suggestion
    logger.info(f"Vaccine Center {vaccine_center}: looking for an appointment")
```